src/engine/init.c

```diff
@@ -1149,8 +1149,9 @@ parse(int argc, char **argv)
 			break;
 		case 'T':
 			rc = arg_strtoul(optarg, &dss_storage_tiers, "\"-T\"");
-			if (dss_storage_tiers < 1 || dss_storage_tiers > 4) {
-				printf("Requires 1 to 4 tiers\n");
+			// add a new tier for QLC NVMe SSD
```

Comment:
ok, will consider about this. I added a comment here just because this change point is not so clear, I found it not in the coding stage, just found it when I tried to do some test.